scripts/prisma-version.sh

```diff
@@ -4,8 +4,8 @@ set -eu
 
 CHANNEL="$1"
 
-# TODO: remove this if-condition once we move to dev
-if [ "$CHANNEL" = "dev" ]; then
+# TODO: remove this if-condition once we move to insider
+if [ "$CHANNEL" = "insider" ]; then
     CHANNEL="alpha"
 fi
```

Comment:
I am not sure if this change is correct - this probably should still refer to `dev`

Reply:
I thought the renaming was about the version name too, therefore I removed all mentions of `dev`.

Reply:
I thought this one would refer to the npm channel - but not sure - @divyenduz should know.

Reply:
This should still be called "dev"